source/features/open-all-notifications.tsx

```diff
@@ -136,10 +136,11 @@ void features.add(import.meta.url, {
 	],
 	shortcuts: {
 		p: 'Open selected notifications',
+		gn: 'Open all notifications',
+		gu: 'Open all unread notifications',
```

Comment:
These are just for documentation, they do nothing.

To implement the feature you have to add `data-hotkey` on `button` and update the label `Open all unread notifications from this repo`

The feature must work as:

1. open https://github.com/notifications in a new tab
2. press `g u`

Pressing the buttons on other pages is not what was being asked